pom.xml

```diff
@@ -67,6 +67,8 @@
         <slf4j.version>1.7.25</slf4j.version>
         <zkclient.version>0.10</zkclient.version>
         <zookeeper.version>3.4.10</zookeeper.version>
+        <checkstyle.config.location>checkstyle/checkstyle.xml</checkstyle.config.location>
+        <checkstyle.suppressions.location>checkstyle/common-suppressions.xml</checkstyle.suppressions.location>
```

Comment:
in this case i think we want to separate these out. we actually already have these named differently in children -- they are named just `checkstyle/suppressions.xml` in children projects. e.g. https://github.com/confluentinc/kafka-connect-hdfs/blob/master/checkstyle/suppressions.xml I think we've already been overriding them manually without much complaint, e.g. https://github.com/confluentinc/kafka-connect-hdfs/blob/master/checkstyle/suppressions.xml but including a property for it wouldn't hurt.

Reply:
Right, but overriding is a bit more difficult now since you also have to do it for the jenkins profile.